spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -95,7 +95,7 @@
     end
 
     it "renders explorer based on RBAC access to feature 'provider_foreman_add_provider'" do
-      set_user_privileges user_with_feature %w(provider_foreman_add_provider)
+      login_as user_with_feature %w(provider_foreman_add_provider)
```

Comment:
We should now use `login_as` `user_with_feature` to test specific features.  We don't want to stub things when we're testing specific features as OTHER features would get that stubbed response.